web/src/components/Upstream/typings.d.ts

```diff
@@ -19,7 +19,11 @@
  * Schema: https://github.com/apache/apisix/blob/master/apisix/schema_def.lua#L335
  */
 declare namespace UpstreamComponent {
-  type ActiveCheck = {};
+  type ActiveCheck = {
+    url?: string;
+    host?: string;
+    http_path?: string;
```

Comment:
IMO, this is to merge two input boxes into one, right?  so why are there two fields, http_path and host?

Reply:
I think it may be that, when editing, the interface should still return two fields 🤔

Reply:
Make sense.